src/data/finset/slice.lean

```diff
@@ -0,0 +1,103 @@
+/-
+Copyright (c) 2021 Bhavik Mehta, Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta, Alena Gusakov, Yaël Dillies
+-/
+import data.fintype.basic
+import order.antichain
+
+/-!
+# `r`-sets and slice
+
+This file defines the `r`-th slice of a set family and provides a way to say that a set family is
+made of `r`-sets.
+
+An `r`-set is a finset of size `r`. The `r`-th slice of a set family is the set family made of its
```

Comment:
I think it would be good to mention that you're using "size" as a synonym for cardinality.
```suggestion
An `r`-set is a finset of cardinality `r` (or *size* `r`). The `r`-th slice of a set family is the set family made of its
```

Do you want to use backticks for `r` everywhere in the docstring rather than latex? It's fine in subsequent sections, but in this more mathy intro section I'd suggest using latex to make it clearer that `r` is meant to be a mathy variable.

Reply:
My point was to explain why it was called `sized`, but I'm happy to stick with cardinality. However I like having backticks for such simple things (aren't all variables mathy? 🤔).

Reply:
The parenthetical "or *size* `r`" is one way to explain that -- I actually wasn't sure before I read the Lean definition of `sized` that it was the sets of cardinality `r`, hence the suggestion. Maybe you'll like this wording more:
```
An `r`-set is a finset of cardinality `r` (what we also call a finset of *size* `r`).
```

Reply:
Whatabout now?